docs/Dockerfile

```diff
@@ -27,12 +27,9 @@ RUN conda install -c conda-forge --file ./docs/requirements.txt -y
 
 RUN pip install .
 
-# RUN pip install lavavu-osmesa==1.8.45 
-ENV LD_LIBRARY_PATH=/opt/conda/lib/python3.10/site-packages/lavavu_osmesa.libs
-
 ENV TINI_VERSION v0.19.0
```

Comment:
If we aren't using lavavu why is tini still there?

Reply:
From what I gathered, tini is good practice for docker init, to manage zombie processes & signal forward processes. As I am not the original author of this code, I wasn't aware that this was here because of lavavu... 
Removed in https://github.com/Loop3D/map2loop/pull/71/commits/89fd60f35445f96894c153d3f8c5f6451bcbf0ed